### ISSUE TITLE
SelectManyMenu: Accessibility issues aria-label for Options List

### ISSUE THREAD

**Issue by pfumi** [primefaces/primefaces#13848]
### Describe the bug

axe-core raises accessibility issues for the SelectManyMenu component.

![Image](https://github.com/user-attachments/assets/8a628f25-58a4-4d77-ae3b-5b82a43c49c5)

### Reproducer

1. Run [primefaces-test-issue13848.zip](https://github.com/user-attachments/files/20443876/primefaces-test-issue13848.zip)
2. The list of options has no label

### Expected behavior

Place a generic aria-label on the `<ul>`. E.g. Primevue sets "Option List" for the AutoComplete -> https://primevue.org/autocomplete/

### PrimeFaces edition

Community

### PrimeFaces version

15.0.3

### JSF implementation

All

### JSF version

4.0

### Java version

11

**Comment by pfumi** [comment]
testet and looks fine 👍
